lib/vsc/utils/frozendict.py

```diff
@@ -17,14 +17,20 @@
 # IN THE SOFTWARE
 
 import operator
-from UserDict import DictMixin
-
+import sys
+from distutils.version import LooseVersion
+pyver = sys.version.split(' ')[0]
+if LooseVersion(pyver) < LooseVersion('3.0'):
+    from UserDict import DictMixin as MutableMapping
+else:
+    # see http://stackoverflow.com/questions/11165188/how-to-achieve-the-functionality-of-userdict-dictmixin-in-python-3
+    from collections import MutableMapping
```

Comment:
this ugly check shouldn't be here at all, collections.MutableMapping is available in python 2.6, yes, everything is easier when you drop python 2.4 and 2.5 ;-)